### ISSUE TITLE
No examples??

### ISSUE THREAD

**Issue by Malkitti** [elvinpoon/tensorflow-CWS-LSTM#1]
There is no examples. There is no  description.

**Comment by elvinpoon** [comment]
I'm not sure what u are looking for? Do you mean train files for 'example'?
PS: I've upload word embedding files and train file for u

**Comment by Malkitti** [comment]
Yeah.  A basic example shows how it works.  A small description in English about the model or the papers that are implemented.  I understand that this code is about using windowed LSTMs for  Chinese word segmentation?
